code/game/objects/items/storage/dufflebags.dm

```diff
@@ -22,6 +22,7 @@
 
 /obj/item/storage/backpack/duffelbag/Initialize(mapload)
 	. = ..()
+	slowdown += zip_slowdown // BUBBER FIX
```

Comment:
```suggestion
	slowdown += zip_slowdown
```

Since we're mirroring it, leave it in the format TG committed it so it doesn't conflict down the line.